tst/regression/regression_test.py

```diff
@@ -208,10 +208,13 @@ def gold_comparison(variables, input_file, modified_inputs={},
     modify_input(key, modified_inputs[key], TEMPORARY_INPUT_FILE)
 
   # Run test problem
+  preamble = []
+  if use_mpiexec:
+      preamble = preamble + ['mpiexec', '-n', '1']
```

Comment:
`mpiexec` is optional now.

Reply:
We could do something like that but then all of a sudden we'd need per-machine configurations. Maybe something to revisit in phoebus if we have time for firelink in the future.